client/src/components/Verejne/entityHelpers.js

```diff
@@ -0,0 +1,16 @@
+// @flow
+import type {Entity} from '../../state'
+
+// entity which is a number is individual, district starts with letter
+export const isIndividual = (eid: string): boolean => {
+  return '0123456789'.indexOf(eid[0]) === -1
+}
+
+export const isPolitician = (entity: Entity): boolean => {
+  // TODO: == is on purpose as this field can be: null, 0, true...
+  return entity.ds && entity.ds[0] == 1
+}
+
+export const hasContractsWithState = (entity: Entity): boolean => {
+  return entity.ds && isIndividual(entity.eid) && entity.ds[1] > 0
```

Comment:
are you sure ds has at least 2 chars ? and why checking > 0 ? (at least add a comment regarding the second question to the code)